### PR TITLE
Moving default channel list to default flag value

### PR DESCRIPTION
**Pull request by poppyseedplehzr**
This PR Address #2980 by changing the hard coded channels we automatically subscribed to into default flag values. This should allow folks to subscribe to as many _or as few_ channels as they'd like.

### CODE REVIEW

File: osquery/tables/events/windows/windows_events.cpp [hunk 1]
```diff
@@ -26,34 +26,25 @@ namespace pt = boost::property_tree;
...
-* --windows_additional_event_channels flag.
+* channels specify them via this flag as a comma separated list.
 */
-const std::set<std::wstring> kDefaultSubscriptionChannels = {
-    L"System", L"Application", L"Setup", L"Security",
-};
+FLAG(string,
+     windows_event_channels,
+     "System,Application,Setup,Security",
+     "Comma-separated list of additional Windows event log channels");
```
Comment (theopolis) [inline osquery/tables/events/windows/windows_events.cpp]: Might want to nix the "additional" in the description.

Comment (poppyseedplehzr) [inline osquery/tables/events/windows/windows_events.cpp]: Nice catch, thanks.

**Comment by theopolis** [comment]
Does this need a documentation change in the CLI-flags page? http://osquery.readthedocs.io/en/stable/installation/cli-flags/

**Comment by poppyseedplehzr** [comment]
@theopolis yeah that would probably be a good idea, will update now :)

**Pull request merged**